src/app/calendar/clientPage.tsx

```diff
@@ -0,0 +1,41 @@
+"use client";
+
+import React from "react";
+import Calendar, { CalendarEvent as CalendarEvent } from "../../components/Calendar/Calendar";
+import { styled } from "styled-components";
+
+const SampleCalendar: React.FC<{}> = () => {
+    const sampleEvents: CalendarEvent[] = [
+        {
+            id: "a",
+            title: "event1",
+            start: new Date("2023-07-11"), // default Date constructor assumes UTC by default
+            end: new Date("2023-07-12"),
+            backgroundColor: "red",
+        },
+        {
+            id: "b",
+            title: "event2",
+            start: new Date("2023-07-12"),
+            end: new Date("2023-07-13"),
+            daysOfWeek: [3],
+            startRecur: "2023-07-12",
+            endRecur: "2023-08-25",
+            borderColor: "yellow",
+        },
+        {
+            id: "c",
+            title: "event3",
+            start: new Date("2023-07-13T12:30:00"),
+            textColor: "pink",
+        },
```

Comment:
The text colour doesn't actually change on this event - it's still black. Also, it seems to be rendered different than the other events. Maybe because it doesn't have a specified end date?

Edit: Just realised it's because some events are "All Day" events, and others are not

Reply:
yes, but we have now changed end date to be required.

Reply:
the text colour and border colour are also there from what we can see. Can you try again to see if it works? (it might not be very easy to see).